cfpb/sbl-frontend
# fix: Style misalignments and content updates

Closes #375 

## Changes

- Institution profile: Update email subject (Shared between View/Update pages)
- User profile: Apply standard width restriction for section introductions
- Filing home: Hero text width now at 670px wide
- Filing overvview: Sentence case
- Complete user profile form: Label `Additional details` as optional
- Complete user profile form: Add LEI helper text
- Summary error Alert: Add link underline styling

## How to test this PR

1. Email subject - Visit an [Insitution profile](http://localhost:8899/institution/123456789TESTBANK123) and click on the `email our support staff` link.  Verify subject line
2. User profile - Inspect section headers and confirm their content is restricted to 670px wide
3. Filing homepage hero - Inspect hero and confirm it's content is restricted to 670px wide
4. Visit Filing overview (/filing) - Verify sentence case

## Screenshots
### Email subject line
<img width="659" alt="image" src="https://github.com/cfpb/sbl-frontend/assets/2592907/c330414e-38f5-4578-8027-b8f5ef191c9a">

### User profile
<img width="764" alt="image" src="https://github.com/cfpb/sbl-frontend/assets/2592907/bdacb825-3761-41ad-a5ad-c1c330f7d7e9">

### Filing homepage hero
<img width="701" alt="Screenshot 2024-04-04 at 10 01 09 AM" src="https://github.com/cfpb/sbl-frontend/assets/2592907/54195066-f40c-4cf9-b008-dd5db3935753">

### Filing overview - Sentence case
<img width="811" alt="image" src="https://github.com/cfpb/sbl-frontend/assets/2592907/3bc9d57b-6ac6-4da7-b48a-be6d321a107b">

### Complete user profile - Additional details - Optional
![screencapture-localhost-8899-profile-complete-2024-04-04-17_06_02](https://github.com/cfpb/sbl-frontend/assets/2592907/ac3a1c7c-176f-4db9-b474-aec1ac9dba2c)

### Complete user profile - Add LEI helper text
![screencapture-localhost-8899-profile-complete-2024-04-04-17_14_24](https://github.com/cfpb/sbl-frontend/assets/2592907/86ffcf04-6f59-48f7-a6fa-162ff0fc8e13)
